views/transaction.pug

```diff
@@ -15,6 +15,29 @@ block content
 			small.text-monospace #{txid}
 		hr
 
+		if (slpValidResponse && txDetailsResponse.tokenReceivers)
+			h1.h3.word-wrap SLPA Transaction			
+				br
+				if (txDetailsResponse && txDetailsResponse.transactionType === "SEND")
+					small.text-monospace Token ID: #{txDetailsResponse.tokenIdHex}
+					br
+					small.text-monospace Token Name: #{txDetailsResponse.tokenName} (#{txDetailsResponse.tokenTicker})
+					if (txDetailsResponse.tokenSendingAddress)						
+						br
+						br
+						small.text-monospace #{txDetailsResponse.tokenSendingAddress} sent #{txDetailsResponse.tokenReceivers[0].amount} #{txDetailsResponse.tokenName} (#{txDetailsResponse.tokenTicker}) to #{txDetailsResponse.tokenReceivers[0].address}
+					else
+						br
+						br
+						small.text-monospace #{txDetailsResponse.tokenReceivers[0].address} received #{txDetailsResponse.tokenReceivers[0].amount} #{txDetailsResponse.tokenName} (#{txDetailsResponse.tokenTicker})
+				if (txDetailsResponse && txDetailsResponse.transactionType === "GENESIS")
+					small.text-monospace Genesis Transaction
+					br
+					small.text-monospace Token ID: #{txDetailsResponse.tokenIdHex}
+					br
+					small.text-monospace Token Name: #{txDetailsResponse.tokenName} (#{txDetailsResponse.tokenTicker})
+		hr
+
```

Comment:
this looks like we forgot a debug printf in production code. maybe the best thing would be to have a separate template for SLP transactions only? until we know, perhaps just a simple unit test for the new routes is best.